client/src/components/relatedProducts/card.jsx

```diff
@@ -2,6 +2,14 @@
 import React from 'react';
 import styled from 'styled-components';
 import TestStarIcon from '../stars/star.jsx'
+//////////
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCircleXmark } from '@fortawesome/free-solid-svg-icons'
+import {faStar} from '@fortawesome/free-solid-svg-icons'
+
+let second= {
+  color: 'cornflowerblue',
+}
```

Comment:
You can import the different icons on the same line, like 
import { faCircleXmark, faStar } from '@fortawesome/free-solid-svg-icons'

Reply:
good to know, thanks. will fix that.